albumentations/augmentations/functional.py

```diff
@@ -32,6 +32,16 @@ def wrapped_function(img, *args, **kwargs):
     return wrapped_function
 
 
+def angle_to_2pi(angle):
```

Comment:
We should have a docstring here. 

Or name to be slightly longer like `angle_to_2pi_range`